### PR TITLE
Issue 225

### PR DESCRIPTION
**Pull request by euliancom**
# Description
Added `output.id` as `output.thub_id` and kept original `output.id` in the `output.tf` file from aws templates.
Updated all `README.md` files from aws templates.

Fixes # (issue)

## Type of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a change to the documentation

## How Has This Been Tested?

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have checked that my changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/templates/aws/acm_certificate/README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ It's recommended to specify create_before_destroy = true in a lifecycle block to
 | Name | Description | Type |
 |------|-------------|:----:|
 |id|The ARN of the certificate|string|
```
Comment (eistrati) [inline src/templates/aws/acm_certificate/README.md]: Why does it say `ARN` instead of `ID`?

File: src/templates/aws/acm_certificate/README.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ It's recommended to specify create_before_destroy = true in a lifecycle block to
...
+|thub_id|The ARN of the certificate|string|
```
Comment (eistrati) [inline src/templates/aws/acm_certificate/README.md]: Why does it say `ARN` instead of `THUB_ID`?

File: src/templates/aws/alb/README.md [hunk 3]
```diff
@@ -28,6 +28,7 @@ Provides a Load Balancer resource.
 | Name | Description | Type |
 |------|-------------|:----:|
 |id|The ARN of the load balancer (matches arn).|string|
```
Comment (eistrati) [inline src/templates/aws/alb/README.md]: Why does it say `ARN` instead of `ID`?

File: src/templates/aws/alb/README.md [hunk 4]
```diff
@@ -28,6 +28,7 @@ Provides a Load Balancer resource.
...
+|thub_id|The ARN of the load balancer (matches arn).|string|
```
Comment (eistrati) [inline src/templates/aws/alb/README.md]: Why does it say `ARN` instead of `THUB_ID`?

**Comment by euliancom** [comment]
@eistrati can you check, please.

**Pull request merged**